src/source/Signaling/Signaling.c

```diff
@@ -37,6 +37,8 @@ STATUS createSignalingSync(PSignalingClientInfoInternal pClientInfo, PChannelInf
     CHK_STATUS(validateSignalingCallbacks(pSignalingClient, pCallbacks));
     CHK_STATUS(validateSignalingClientInfo(pSignalingClient, pClientInfo));
 
+    pSignalingClient->version = 1;
```

Comment:
Let us replace 1 with this to align with rest of the SDK:

`#define SIGNALING_CLIENT_CURRENT_VERSION      1`
Also, I dont see this being used anywhere. Since it is an internal structure, I dont see the significance of setting a version at the moment.

Reply:
Made these changes in the latest commit